### PR TITLE
Ems logger integration (refact of flash messages)

### PR DESCRIPTION
**Pull request by theus77**

### CODE REVIEW

File: Controller/Views/CalendarController.php [hunk 1]
```diff
@@ -26,16 +33,16 @@ public function updateAction(View $view, Request $request)
                 $field = $view->getContentType()->getFieldType()->__get('ems_'.$view->getOptions()['dateRangeField']);
                 
 
-                /**@var \DateTime $from */
-                /**@var \DateTime $to */
-                $from = new \DateTime($request->request->get('start', false));
+                /**@var DateTime $from */
+                /**@var DateTime $to */
```
Comment (dameert) [inline Controller/Views/CalendarController.php]: the @var definition should not be necessary when the type is the same as the created Class.
+ it won't work anyway if there is no space between /** @var

File: Command/LockCommand.php [hunk 2]
```diff
@@ -56,21 +53,24 @@ protected function configure()
     }
 
     /**
-     * @inheritdoc
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return int|void|null
+     * @throws Exception
```
[the remark this answers is not in the document]
Comment (theus77) [reply Command/LockCommand.php]: Ok, I got it (still to decide about the throws

File: Controller/AuthTokenController.php [hunk 3]
```diff
@@ -2,17 +2,16 @@
...
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
 
 class AuthTokenController extends AppController
 {
     /**
-     * @Route("/auth-token", name="auth-token", defaults={"_format": "json"})
-     * @Method({"POST"})
+     * @param Request $request
```
[the remark this answers is not in the document]
Comment (theus77) [reply Controller/AuthTokenController.php]: ok

File: Controller/AppController.php [hunk 4]
```diff
@@ -180,17 +172,26 @@ protected function getHelperService()
...
 
+    /**
+     * @param string $fieldTypeNameOrServiceName
+     * @return DataFieldType
+     * @throws ElasticmsException
+     */
     protected function getDataFieldType(string $fieldTypeNameOrServiceName): DataFieldType
     {
-        return $this->formRegistry->getType($fieldTypeNameOrServiceName)->getInnerType();
+        $dataFieldType = $$this->formRegistry->getType($fieldTypeNameOrServiceName)->getInnerType();
```
[the remark this answers is not in the document]
Comment (theus77) [reply Controller/AppController.php]: fixed

File: Controller/ContentManagement/DataController.php [hunk 5]
```diff
@@ -293,29 +309,38 @@ public function viewDataAction($environmentName, $type, $ouuid)
...
-            'revisionId' => $revision->getId(),
-        ]);
     }
 
     /**
      * @Route("/public-key" , name="ems_get_public_key")
+     * @return Response
      */
-    public function publicKey()
+    public function publicKey():Response
```
[the remark this answers is not in the document]
Comment (theus77) [reply Controller/ContentManagement/DataController.php]: fixed

File: Controller/ContentManagement/CrudController.php [hunk 6]
```diff
@@ -172,18 +207,22 @@ public function deleteAction($ouuid, ContentType $contentType, Request $request)
         try {
             $this->getDataService()->delete($contentType->getName(), $ouuid);
             try {
-                $revision = $this->getDataService()->getNewestRevision($contentType->getName(), $ouuid);
-            } catch (\Exception $exception) {
+                $this->getDataService()->getNewestRevision($contentType->getName(), $ouuid);
```
[the remark this answers is not in the document]
Comment (theus77) [reply Controller/ContentManagement/CrudController.php]: cleaned

**Pull request merged**